external/storm-druid/README.md

```diff
@@ -0,0 +1,143 @@
+# Storm Druid Bolt and TridentState
+
+This module provides core Storm and Trident bolt implementations for writing data to [Druid] (http://druid.io/) data store.
```

Comment:
@omkreddy Remove space between [] and () to make MD generate a hyperlink. Currently it shows as text. Same with other places in this doc.